main/src/ca/uwaterloo/flix/language/phase/unification/Substitution.scala

```diff
@@ -218,4 +221,46 @@ case class Substitution(m: Map[Symbol.TypeVarSym, Type]) {
     Substitution(m2)
   }
 
+  /**
+    * Prioritizes the given symbol so that it is not replaced by the substitution.
+    *
+    * This is done by:
+    * * rigidifying the variable `x` (so that it cannot appear on the left of the unification result)
+    * * unifying `x` with its replacement `this(x)` to make a new substitution `s`
+    * * putting `s` into the `this` substitution
+    */
+  def prioritize(sym0: Symbol.KindedTypeVarSym)(implicit flix: Flix): Substitution = {
```

Comment:
Computes an equ-most general substitution with the given type variable symbol as `rigid`. 

That is, the resulting subst has `sym = sym`.